### PR TITLE
dependency bump

### PR DESCRIPTION
**Pull request by mversic**

### CODE REVIEW

**Comment by Mingela** [comment]
I'd say this new code style change appending `,` to every last parameter is excessive

**Comment by Mingela** [comment]
or at least place this kind of change to a separate commit

**Comment by mversic** [comment]
> Kotlin style guide:

Kotlin style guide says:

> Trailing commas are entirely optional – your code will still work without them. The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it at your discretion for the call site.

but I can find the flag to custom configure this rule in ktlint for us

File: modules/block/build.gradle [hunk 1]
```diff
@@ -7,7 +7,3 @@ dependencies {
     testImplementation "org.jetbrains.kotlin:kotlin-test-junit5:$kotlinVer"
     testImplementation "org.jetbrains.kotlin:kotlin-test:$kotlinVer"
 }
-
-kotlinter {
-    disabledRules = ['no-trailing-spaces']
```
Comment (arndey) [inline modules/block/build.gradle:12]: why? @mversic

File: build.gradle [hunk 2]
```diff
@@ -70,23 +74,18 @@ subprojects {
...
     compileTestKotlin {
         kotlinOptions {
             freeCompilerArgs = ["-Xjsr305=strict"]
-            jvmTarget = 1.8
         }
     }
 
     kotlinter {
         ignoreFailures = false
-        indentSize = 4
```
Comment (arndey) [inline build.gradle:86]: explain please @mversic

Comment (mversic) [reply build.gradle:86]: the parameter is no longer supported by kotlinter

File: modules/block/build.gradle (hunk 1 above)
Comment (mversic) [reply modules/block/build.gradle:12]: it is no longer supported by kotlinter

File: modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/SchemaReader.kt [hunk 3]
```diff
@@ -16,7 +17,7 @@ class SchemaReader {
     fun readSchema(fileName: String): Map<String, Any> {
         val resource = Thread.currentThread().contextClassLoader.getResourceAsStream(fileName)!!
         val sb = StringBuilder()
-        val lines = resource.bufferedReader().lines().toList()
+        val lines = resource.bufferedReader().lines().collect(Collectors.toList<String>())
```
Comment (arndey) [inline modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/SchemaReader.kt:20]: smells like Java, why so?

File: modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/TypeResolver.kt [hunk 4]
```diff
@@ -254,11 +256,13 @@ object EnumResolver : Resolver<EnumType> {
                 EnumType.Variant(
                     (it["tag"] ?: throw IrohaSdkException("Enum name not found")) as String,
                     (it["discriminant"] ?: throw IrohaSdkException("Enum discriminant not found")) as Int,
-                    variantProperty?.let(schemaParser::createAndGetNest)
+                    variantProperty?.let(schemaParser::createAndGetNest),
                 )
             }
             EnumType(name, generics, variants)
-        } else null
+        } else {
```
Comment (arndey) [inline modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/TypeResolver.kt:263]: were these brackets added automatically by auto-formatting?

File: modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/codec/IntMax.kt [hunk 5]
```diff
@@ -10,7 +10,8 @@ enum class IntMax(val max: BigInteger) {
     U128(BigInteger.valueOf(2).pow(128) - BigInteger.ONE),
     U256(BigInteger.valueOf(2).pow(256) - BigInteger.ONE),
     I128(BigInteger.valueOf(2).pow(127) - BigInteger.ONE),
-    I256(BigInteger.valueOf(2).pow(255) - BigInteger.ONE);
+    I256(BigInteger.valueOf(2).pow(255) - BigInteger.ONE),
```
Comment (arndey) [inline modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/codec/IntMax.kt:13]: omg, even here

File: modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt [hunk 6]
```diff
@@ -53,7 +53,7 @@ public sealed class ConfigurationEvent : ModelEnum {
      * 'Created' variant
```
Comment (arndey) [inline modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt:53]: we don't edit `generated` module entities btw

**Review by arndey — CHANGES_REQUESTED** [review]
some comments

File: modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt (hunk 6 above)
Comment (mversic) [reply modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt:53]: ok, that makes sense

File: modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/TypeResolver.kt (hunk 4 above)
Comment (mversic) [reply modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/TypeResolver.kt:263]: yes

File: modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt (hunk 6 above)
Comment (arndey) [reply modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt:53]: we make changes in `codegen`

File: modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/SchemaReader.kt (hunk 3 above)
Comment (mversic) [reply modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/SchemaReader.kt]: hm, I don't remember why I would change this, restored now

File: modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt (hunk 6 above)
Comment (mversic) [reply modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt]: fixed, check it now

File: modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/codec/IntMax.kt (hunk 5 above)
Comment (mversic) [reply modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/codec/IntMax.kt:13]: I know, but I'd rather not fight the linter

**Review by Mingela — APPROVED** [review]

**Review by arndey — APPROVED** [review]

**Pull request merged**